project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 import Keys._
 
 object Dependencies {
-  lazy val http4sVersion = "0.18.19"
+  lazy val http4sVersion = "0.19.0"
```

Comment:
The currently version is `0.19.0-M4`, 0.19.0 was mistagged and should be avoided if possible.